libs/designer/src/lib/core/utils/parameters/helper.ts

```diff
@@ -861,7 +861,7 @@ export function loadParameterValue(parameter: InputParameter): ValueSegment[] {
 
   if (parameter.isNotificationUrl) {
     valueObject = `@${constants.HTTP_WEBHOOK_LIST_CALLBACK_URL_NAME}`;
-  } else if (parameter.value) {
+  } else if (parameter.value || (parameter.type === 'boolean' && parameter.value === false)) {
```

Comment:
NIT: I wonder if we can use the `isBoolean` function and send the parameter.value.

Reply:
ok! made that change